### PR TITLE
Update macos-development.md

### PR DESCRIPTION
**Pull request by TheWerewolf**
It took me two weeks and a lot of painful floundering around to realise this is how you take an AvalonUI app project and run it on MacOS because I've also been looking at Uno.Platform, OpenSilver and Xamarin.Forms.

These 10 or so sentences would have saved me SO much time and frustration.

## What does the pull request do?

**Scope of this PR:**
- [x] fix or update to an existing page
- [ ] add a new page to the docs

## Checklist

**If this is a new Page**
- [ ] Added the new page to [Summary.md](https://docs.gitbook.com/getting-started/git-sync/content-configuration#summary)

**In any case**
- [x] Spell-checking done
- [x] Checked if all hyperlinks work
- [x] Checked if all images are visible

### CODE REVIEW

**Comment by TheWerewolf** [comment]
I am unaware of how to agree to the CLA, but the change is not code related, rather just providing additional help for new devs to build macOS versions of their AvaloniaUI project

**Comment by TheWerewolf** [comment]
@dotnet-policy-service agree

**Review by Arhell — COMMENTED** [review]
@maxkatz6

**Comment by TheWerewolf** [comment]
@dotnet-policy-service agree company=”VargrSoft”

File: guides/developer-guides/macos-development.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ description: Getting started developing for the macOS backend
...
+
+It's actually really simple. If you're developing in Windows (which is easier for a Windows developer), once your AvaloniaUI application is looking about right and you're ready to try it on macOS, copy your entire project folder to your Mac (don't worry - all the files will be treated as basic flat files). If you have a github for it, commit it and then check it out on your dev Mac machine.
+
+You'll need Visual Studio for MacOS. 
+
+Open your project's SLN file in Visual Studio for MacOS.
+
+Then run.
+
+That's it.
```
Comment (maxkatz6) [inline guides/developer-guides/macos-development.md:17]: Strictly speaking, you don't need VS for Mac.
You can use Rider, but you don't need it either.
You just need "dotnet run".

**Comment by TheWerewolf** [comment]
Valid point. I use VS exclusively, but I assume the basic concept is still valid. Let make an amendment.

File: guides/developer-guides/macos-development.md (hunk 1 above)
Comment (TheWerewolf) [reply guides/developer-guides/macos-development.md]: Added additional details for JetBrains and command line users.

**Pull request merged**